app/views/idv/please_call/show.html.erb

```diff
@@ -1,11 +1,14 @@
-<% content_for(:pre_flash_content) do %>
-  <%= render StepIndicatorComponent.new(
-        steps: Idv::StepIndicatorConcern::STEP_INDICATOR_STEPS,
-        current_step: :secure_account,
-        locale_scope: 'idv',
-        class: 'margin-x-neg-2 margin-top-neg-4 tablet:margin-x-neg-6 tablet:margin-top-neg-4',
-      ) %>
-<% end %>
+<% if @in_person == false or @in_person.nil? %>
```

Comment:
Is this a case for [blank?](https://apidock.com/rails/Object/blank%3F), I wonder? (It's not identical, because `@in_person = ' '` would count as `blank?`, but that's not a sensible value in this case.)

This is an unimportant, completely optional suggestion.

Reply:
Yeah `bkank?` could be a good option here, more concise certainly. Looks like it does handle whitespace strings?

Reply:
But this value is also a boolean, and not a string. So I think maybe we leave the logic check as is for now?